index.js

```diff
@@ -1,23 +1,193 @@
-console.log("hello there")
-
-var test = {
-	escape: function(html) {
-		return String(html)
-			.replace(/&/g, '&amp;')
-			.replace(/"/g, '&quot;')
-			.replace(/'/g, '&#39;')
-			.replace(/</g, '&lt;')
-			.replace(/>/g, '&gt;');
-	},
-
-	unescape: function(html) {
-		return String(html)
-			.replace(/&amp;/g, '&')
-			.replace(/&quot;/g, '"')
-			.replace(/&#39;/g, '\'')
-			.replace(/&lt;/g, '<')
-			.replace(/&gt;/g, '>');
-	}
+
+// placeholder for now - should talk when merged to properly integrate this with 
+// existing "board" and "players" implementations
+function Board(players) { 
+  this.players = players; // just a placeholder for now - on merge should combine 
+  this.spaces = [];
+} 
+
+// class definitions for any space on the board - 
+// properties, utilities, free parking, go, etc. 
+function Space(name) { 
+  this.name = name; 
+  this.occupier = null;
+}
+// every space needs a landing action
+Space.prototype.perform_landing_action = function(){
+  console.log("You landed on " + this.name);
+}; 
+
+function Property(name, cost, property_group) { 
+  Space.call(this, name);
+  this.cost = cost;
+  this.mortage = .5*cost;
+  this.property_group = property_group; // "red", "blue", "railroad", etc.
+  this.owner = null;
+}
+Property.prototype = Object.create(Space.prototype); // subclassing space
+
+// rent should be array with following format: 
+// [1 owned, 2 owned ... (3 owned, 4 owned)]
+function UtilityProperty(name, cost, property_group, rent) { 
```

Comment:
the logic for RR vs utility is pretty different so maybe we want two different subclasses?